### PR TITLE
HCALDQM: Add monitoring for HF TP TDC cut (10_1_X)

### PR DESCRIPTION
**Pull request by DryRun**
Adds monitoring plots for HF TP TDC cut: efficiency of cut vs. ieta vs iphi and ieta. Also adds the numerator and denominator of the efficiency calculation (i.e. data TP occupancy / emulated TP occupancy with no TDC cut).

### CODE REVIEW

**Comment by jfernan2** [comment]
please test

**Comment by DryRun** [comment]
runTheMatrix failed due to the new digi collection (emulated TPs with no TDC cut) was only added in the online configurations, not the offline. We only want these plots in the online DQM, so I'll add a condition that loads the collection only in online.

**Comment by thomreis** [comment]
please test

**Comment by DryRun** [comment]
Sorry for the sloppiness - I fixed the problem so that the changes should be invisible to offline DQM (intended only for online), so I think the RelVals should work now.

File: DQM/HcalTasks/plugins/HcalOnlineHarvesting.cc [hunk 1]
```diff
@@ -138,6 +138,33 @@ HcalOnlineHarvesting::HcalOnlineHarvesting(edm::ParameterSet const& ps) :
...
+		MonitorElement* meOccupancy_HF_ieta = ig.get("Hcal/TPTask/OccupancyDataHF_ieta/OccupancyDataHF_ieta");
+		MonitorElement* meOccupancyNoTDC_HF_ieta = ig.get("Hcal/TPTask/OccupancyEmulHFNoTDC_ieta/OccupancyEmulHFNoTDC_ieta");
+
+		if (meOccupancy_HF_depth && meOccupancyNoTDC_HF_depth && meOccupancy_HF_ieta && meOccupancyNoTDC_HF_ieta) {
+			TH2F* hOccupancy_HF_depth = meOccupancy_HF_depth->getTH2F();
+			TH2F* hOccupancyNoTDC_HF_depth = meOccupancyNoTDC_HF_depth->getTH2F();
+			TH1F* hOccupancy_HF_ieta = meOccupancy_HF_ieta->getTH1F();
+			TH1F* hOccupancyNoTDC_HF_ieta = meOccupancyNoTDC_HF_ieta->getTH1F();
+
+			TH2F *hEfficiency_HF_depth = (TH2F*)hOccupancy_HF_depth->Clone();
```
Comment (dmitrijus) [inline DQM/HcalTasks/plugins/HcalOnlineHarvesting.cc]: You must delete what you get from Clone()

**Comment by dmitrijus** [comment]
-1

**Comment by dmitrijus** [comment]
+1

**Comment by fabiocos** [comment]
+1

**Pull request merged**